.circleci/config.yml

```diff
@@ -1,100 +1,99 @@
 version: 2
 jobs:
-  test:
-    docker:
-      - image: circleci/python:3.6.1
-      - image: circleci/postgres:9.4
-        environment:
-          - TERM: xterm-256color
-          - POSTGRES_USER: circleci
-          - POSTGRES_DB: circlecidb
-          - POSTGRES_PASSWORD: $DB_PASSWORD
-    working_directory: ~/andela-socials-backend
+    test:
+        docker:
+            - image: circleci/python:3.6.1
+            - image: circleci/postgres:9.4
+              environment:
+                  - TERM: xterm-256color
+                  - POSTGRES_USER: circleci
+                  - POSTGRES_DB: circlecidb
+                  - POSTGRES_PASSWORD: $DB_PASSWORD
+        working_directory: ~/andela-socials-backend
 
-    steps:
-      - checkout
-      - setup_remote_docker
-      - run:
-          name: install dependencies
-          command: |
-            python3 -m venv venv
-            . venv/bin/activate
-            pip install -r server/requirements.txt
-      - run:
-          name: run tests
-          command: |
-            . venv/bin/activate
-            export DJANGO_SETTINGS_MODULE=a_socials.settings.dev
-            python server/manage.py test --snapshot-update server/api/tests/ server/graphql_schemas/tests/
+        steps:
+            - checkout
+            - setup_remote_docker
+            - run:
+                  name: install dependencies
+                  command: |
+                      python3 -m venv venv
+                      . venv/bin/activate
+                      pip install -r server/requirements.txt
+            - run:
+                  name: run tests
+                  command: |
+                      . venv/bin/activate
+                      export DJANGO_SETTINGS_MODULE=a_socials.settings.dev
+                      python server/manage.py test --snapshot-update server/api/tests/ server/graphql_schemas/tests/  --keepdb
```

Comment:
Please remove the `--keep-db` flag. The test database should ideally be deleted.